packages/theme_tailor/lib/src/generator/theme_tailor_generator.dart

```
@@ -178,6 +220,17 @@ class _TailorClassVisitor extends SimpleElementVisitor {
     if (ignoreAnnotationTypeChecker.hasAnnotationOf(element)) return;
 
     if (element.isStatic && element.type.isDartCoreList) {
+      if (!element.isConst) {
+        hasNonConstantElement = true;
+
+        if (requireConstThemes) {
+          print(
+              'Field "${element.name}" needs to be a const in order to be included'
+              ' in generated theme');
```

Actually we should throw an error here since it is required